style/notebookScheduler.css

```diff
@@ -0,0 +1,238 @@
+}
+
+.create-scheduler-form-element {
+  margin-top: 15px;
+}
+
+.create-scheduler-form-element-input-file {
+  margin-top: 15px;
+  display: flex;
+  align-items: center;
+}
+
+.edit-notebook-style {
+  cursor: pointer;
+  margin-top: 10px;
+  padding-left: 15px;
+}
+
+.create-job-scheduler-title {
+  padding: 0px 15px;
+  height: 48px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: var(--jp-ui-font-color0);
+}
+
+.create-scheduler-label {
+  margin-top: 15px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  color: var(--jp-ui-font-color0);
+}
+
+.create-scheduler-style {
+  width: 512px;
+  height: 36px;
+  background: var(--jp-layout-color0);
+  margin-top: 10px;
+  outline: none;
+}
+
+.execution-history-left-wrapper {
+  width: 320px;
+  border: 1px solid var(--jp-layout-color3);
+}
+.execution-history-main-wrapper {
+  display: flex;
+}
+
+.execution-history-right-wrapper {
+  width: calc(100% - 320px);
+  border: 1px solid var(--jp-layout-color3);
+  border-left: none;
+}
+
+.dag-runs-list-table-parent {
+  height: 400px;
+  overflow: auto;
+}
+
+.scheduler-back-arrow-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  padding-left: 10px;
+}
+
+.dag-runs-table-body {
+  overflow-y: auto;
+  max-height: 55vh;
+}
+
+.dag-runs-row-data-parent:hover {
+  background: var(--jp-layout-color2);
+}
+
+.dag-runs-row-data-parent-selected {
+  height: 29px;
+  background: var(--jp-brand-color4);
+  color: var(--jp-brand-color1);
+}
+
+.dag-runs-row-data-parent {
+  background: var(--jp-layout-color0);
+  height: 29px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 18px;
+  color: var(--jp-ui-font-color1);
+  cursor: pointer;
+}
+
+.execution-history-title {
+  display: flex;
+}
+
+.accordion-row-parent {
+  display: flex;
+  width: 100%;
+  height: 30px;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 1px solid var(--jp-layout-color3);
+}
+
+.accordion-row-parent-header {
+  background: rgba(0, 0, 0, 0.04);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  height: 29px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: 18px;
+  color: var(--jp-ui-font-color0);
+  border-bottom: 1px solid var(--jp-layout-color3);
+}
+
+.accordion-row-data {
+  padding-left: 15px;
+  width: 33%;
+}
+
+.logo-row-container {
+  display: flex;
+}
+
+.logo-alignment-style-accordion {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  cursor: pointer;
+}
+
+.logs-content-style {
+  overflow: auto;
+  max-height: 400px;
+  margin: 10px;
+  border-bottom: 1px solid var(--jp-layout-color3);
+}
+
+.accordion-row-data-expand-logo {
+  width: 30px;
+}
+
+.dag-runs-table-data-state-success {
+  background-color: #34a853;
```

Comment:
@Jeyaprakash-NK Should background-color be a variable or is this fine? Please check once.

Reply:
resolved - discussed about this css color code issue